website/docs/d/loadbalancer.html.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: "gridscale"
+page_title: "gridscale: loadbalancer"
+sidebar_current: "docs-gridscale-datasource-loadbalancer"
+description: |-
+  Gets the id of a loadbalancer.
+---
+
+# gridscale_loadbalancer
+
+Get the id of an loabalancer resource.
+
+## Example Usage
+
+```terraform
+data "gridscale_loadbalancer" "foo" {
+	resource_id   = "xxxx-xxxx-xxxx-xxxx"
+}
+```
+
+## Attributes Reference
```

Comment:
We expose all attributes. You can look find them here https://github.com/terraform-providers/terraform-provider-gridscale/blob/master/gridscale/datasource_gridscale_loadbalancer.go#L114